kanidm_book/src/examples/k8s_ingress_example.md

```diff
@@ -6,69 +6,132 @@ Guard your Kubernetes ingress with Kanidm authentication and authorization.
 
 The following are used in this example:
 
-- [Kanidm](installing_the_server.html)
-- [Kubernetes v1.23](https://docs.k0sproject.io/v1.23.6+k0s.2/install/)
+- [Kanidm](../installing_the_server.html)
+- [Kubernetes v1.23 or above](https://docs.k0sproject.io/v1.23.6+k0s.2/install/)
 - [Nginx Ingress](https://kubernetes.github.io/ingress-nginx/deploy/)
+- [A Domain Name](https://domains.google/)
 - [CertManager - With Cluster Issuer](https://cert-manager.io/docs/installation/)
 - [ShellCheck](https://github.com/koalaman/shellcheck#user-content-installing)
```

Comment:
Since we're using it, it should be here:

```suggestion
- [ShellCheck](https://github.com/koalaman/shellcheck#user-content-installing)
- [modem7/docker-starwars](https://github.com/modem7/docker-starwars) - An example web site.
```

Reply:
The surprise is part of the fun :)